lib/rules/no-context.js

```diff
@@ -61,6 +62,19 @@ module.exports = {
     /**
      * @param {ASTNode} node We expect either an ArrowFunctionExpression,
      *   FunctionDeclaration, or FunctionExpression
+     * @returns {Boolean} True if node is a stateless functional component,
+     *   false if not.
+     */
+    function isStatelessComponent(node) {
```

Comment:
`Component.detect` seems not working, so I tried detecting stateless functional component manually. However, `utils.isReturningJSX` didn't work as expected, it's returning `false` no matter what.